### DISCUSSION TITLE
Certificate based ACL

### DISCUSSION THREAD

**Discussion by nerdy-sam** [umurmur/umurmur#202] [Ideas]
What do you think about a login ACL based off of CA signed certs?

**Comment by fmorgner** [comment]
I don't believe this to be duplicate, since @mitzip wants to be able to allow only clients with a cert singed by some specific CA to be able to join the server in the first place, while #35 targets controlling access to certain channels based on the certificate itself.

**Comment by klausenbusk** [comment]
+1
